test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/Tests/ClientSerializerTests.cs

```diff
@@ -133,7 +134,7 @@ public void WriteOneUriOperationParametersToUriShouldReturnUri()
         [TestMethod]
         public void WriteTwoUriOperationParametersToUriShouldReturnUri()
         {
-            DataServiceContext context = new DataServiceContext(new Uri("http://www.odata.org/service.svc"));
+            DataServiceContext context = new DataServiceContext(new Uri("http://www.odata.org/service.svc")).ReConfigureForNetworkLoadingTests();

```

Comment:
You could do this in a setup method for the test class to avoid repeating this boilerplate code. Same for requestinfo and serializer